Libraries/LibWeb/WebAssembly/WebAssembly.cpp

```diff
@@ -157,6 +160,11 @@ namespace Detail {
 
 JS::ThrowCompletionOr<NonnullOwnPtr<Wasm::ModuleInstance>> instantiate_module(JS::VM& vm, Wasm::Module const& module, GC::Ptr<JS::Object> import_object)
```

Comment:
Somewhere in this file, the constructors need declared as native properties on the namespace object:

https://webassembly.github.io/spec/js-api/#error-objects

> Let constructor be a new object, implementing the [NativeError Object Structure](https://tc39.github.io/ecma262/#sec-nativeerror-object-structure), with NativeError set to error.

Reply:
Note that for each `CompileErrorConstructor` and such, step 2 of the construct algorithm needs tweaked.

https://tc39.es/ecma262/#sec-nativeerror

We can't call the current impl of OrdinaryCreateFromConstructor, because CompileErrorPrototype and friends aren't intrinsics from ECMAScript. We'll need to add an overload that calls a getter that gets the prototypes from the Web intrinsics.

Reply:
And adding `CompileError`, `RuntimeError` and `LinkError` to `JS_ENUMERATE_NATIVE_ERRORS` in LibJS/Forward.h is verboten :)